DistFiles/less/common-mixins.less

```diff
@@ -38,4 +38,5 @@
 @MarginOuter: 15mm;
 @MarginBottom: 15mm;
 @ExtraSpaceForBinding: 10mm;
-@MarginInner: @MarginOuter + @ExtraSpaceForBinding;
\ No newline at end of file
+@MarginInner: @MarginOuter + @ExtraSpaceForBinding;
+@MarginEven: (@MarginOuter + @MarginInner) / 2.0; // for when we want the same left and right margins
```

Comment:
When I read this below, I read "even" to mean "opposite of odd". How about "MarginBothSidesSame"